### ISSUE TITLE
Finetuning Harveenchadha/wav2vec2-pretrained-clsril-23-10k on custom data is not working 

### ISSUE THREAD

**Issue by raikarsagar** [Open-Speech-EkStep/vakyansh-wav2vec2-experimentation#17]
Hi,

I am trying to finetune Harveenchadha/wav2vec2-pretrained-clsril-23-10k model with custom Hinglish dataset but traning loss doesnt decrease even after finetuning for about 200epochs. Could you please point out the mistake here? Are we supposed to use any seed for torch initializations in particular for getting consistent results?

**Comment by ghsssq** [comment]
Hi any script that i can run to finetune Harveenchadha/vakyansh-wav2vec2-hindi-him-4200 model and what dataset should i finetune on ?
